templates/python/package/payload.py.twig

```diff
@@ -0,0 +1,96 @@
+import os
```

Comment:
@loks0n I'm pretty sure we need changes in the service.py especially in the functions for createFile, createDeployment and createExecution to use the new payload class. 

Please test these and let me know.